tortoise/__init__.py

```diff
@@ -70,15 +69,15 @@ def _init_relations(cls):
                         raise ConfigurationError(
                             'backward relation "{}" duplicates in model {}'.format(
                                 backward_relation_name, related_model_name))
-                    relation = fields.BackwardFKRelation(model, '{}_id'.format(field))
-                    setattr(related_model, backward_relation_name, relation)
+                    fkrelation = fields.BackwardFKRelation(model, '{}_id'.format(field))
+                    setattr(related_model, backward_relation_name, fkrelation)
                     related_model._meta.filters.update(
-                        get_backward_fk_filters(backward_relation_name, relation)
+                        get_backward_fk_filters(backward_relation_name, fkrelation)
```

Comment:
How about doing it like `fk_relation` and `m2m_relation`? I think it always easier to read it like that, than read concatted words

Reply:
Yes, this is raised more than once. It should go into a project style guide.
That was me changing the variable names to make mypy less confused. I'll fix it now.